code/modules/projectiles/guns/energy.dm

```diff
@@ -799,3 +799,143 @@
+	)
+	flags_gun_features = GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER|GUN_ENERGY|GUN_AMMO_COUNT_BY_SHOTS_REMAINING
+	attachable_offset = list("muzzle_x" = 30, "muzzle_y" = 13,"rail_x" = 6, "rail_y" = 20, "under_x" = 26, "under_y" = 10, "stock_x" = 22, "stock_y" = 12)
+	scatter = 4
+	accuracy_mult_unwielded = 0.7
+	scatter_unwielded = 15
+	recoil_unwielded = 1
+	damage_falloff_mult = 0.5
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/charger/magharness
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/charger/somvet
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness, /obj/item/attachable/gyro)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver
+	name = "\improper Volkite caliver"
+	desc = "Volkite weapons are the pride of Martian weapons manufacturing, their construction being a tightly guarded secret. Infamous for its ability to deflagrate organic targets with its tremendous thermal energy, explosively burning flesh in a fiery blast that can be deadly to anyone unfortunate enough to be nearby. The caliver is the primary rifle of the volkite family, and effective at most ranges and situations."
+	icon_state = "caliver"
+	item_state = "caliver"
+	ammo_level_icon = "caliver"
+	fire_sound = 'sound/weapons/guns/fire/volkite_3.ogg'
+	max_shots = 60
+	ammo_datum_type = /datum/ammo/energy/volkite/medium
+	rounds_per_shot = 36
+	default_ammo_type = /obj/item/cell/lasgun/volkite/highcap
+	allowed_ammo_types = list(
+		/obj/item/cell/lasgun/volkite/highcap,
+		/obj/item/cell/lasgun/volkite/powerpack,
+	)
+	attachable_allowed = list(
+		/obj/item/attachable/reddot,
+		/obj/item/attachable/lasersight,
+		/obj/item/attachable/flashlight,
+		/obj/item/attachable/magnetic_harness,
+		/obj/item/attachable/scope,
+		/obj/item/attachable/scope/mini,
+		/obj/item/attachable/scope/marine,
+		/obj/item/attachable/motiondetector,
+		/obj/item/attachable/buildasentry,
+		/obj/item/attachable/shoulder_mount,
+	)
+	flags_gun_features = GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER|GUN_ENERGY|GUN_AMMO_COUNT_BY_SHOTS_REMAINING
+	attachable_offset = list("muzzle_x" = 38, "muzzle_y" = 13,"rail_x" = 6, "rail_y" = 20, "under_x" = 33, "under_y" = 10, "stock_x" = 22, "stock_y" = 12)
+	accuracy_mult = 1.1
+	aim_slowdown = 0.45
+	wield_delay	= 0.7 SECONDS
+	fire_delay = 0.25 SECONDS
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver/magharness
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver/somvet
+	starting_attachment_types = list(/obj/item/attachable/magnetic_harness, /obj/item/attachable/lasersight)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/caliver/tacsensor
+	starting_attachment_types = list(/obj/item/attachable/motiondetector, /obj/item/attachable/lasersight)
+
+/obj/item/weapon/gun/energy/lasgun/lasrifle/volkite/culverin
+	name = "\improper Volkite culverin"
```

Comment:
```suggestion
	name = "\improper Volkite culverin"
```
Is Volkite supposed to be captizalized or not? It's very inconsistent.

Reply:
lowercase, it's just from me changing things/adding words and forgetting to decapitalise what was originally at the start of the sentence. Or not being clear on whether item names actually NEED capitalisation (looking now I think the answer is no)